lib/Reconciler.spec.lua

```diff
@@ -1,8 +1,87 @@
 return function()
 	local Reconciler = require(script.Parent.Reconciler)
+	local Core = require(script.Parent.Core)
+	local Event = require(script.Parent.Event)
+	local Change = require(script.Parent.Change)
+	local GlobalConfig = require(script.Parent.GlobalConfig)
 
 	it("should reify booleans as nil", function()
 		local booleanReified = Reconciler.reify(false)
 		expect(booleanReified).to.never.be.ok()
 	end)
+
+	describe("tracing", function()
+		GlobalConfig.set({
+			logAllMutations = true
+		})
```

Comment:
This is super flaky. `GlobalConfig` is global for a reason, and if any other tests try to set it, one of them will randomly fail.

In each `it` block, you should set + reset the config value, which also gives you an opportunity to test the flag unset case, too!

Reply:
Oops, yeah. Fixed now!